composer.json

```
@@ -36,7 +36,7 @@
         "friendsofphp/proxy-manager-lts": "^1.0.6",
         "monolog/monolog": "*",
         "phpunit/phpunit": "^9.5.28 || ^10",
-        "predis/predis": "^2.0",
+        "predis/predis": "^3.0",
```

```suggestion
        "predis/predis": "^2.0 || ^3.0",
```

That way the lowest-deps CI builds can ensure compatibility with 2.x still.